src/sysusers/sysusers.c

```diff
@@ -2232,6 +2309,9 @@ static int run(int argc, char *argv[]) {
 #endif
         _cleanup_close_ int lock = -EBADF;
         _cleanup_(context_done) Context c = {
+#if HAVE_AUDIT
+                .audit_fd = -EBADF,
+#endif
```

Comment:
and then introduce a macro AUDIT_CONTEXT_NULL that can be used to initialize the field generically